hack/ci/install-cnv.sh

```diff
@@ -2,12 +2,57 @@
 
 set -ex
 
+CNV_PRERELEASE_VERSION=${CNV_PRERELEASE_VERSION:-}
+
+if [ -z "${CNV_PRERELEASE_VERSION}" ]
+then
+  CNV_RELEASE_CHANNEL=stable
+  CNV_SUBSCRIPTION_SOURCE=redhat-operators
+else
+  CNV_RELEASE_CHANNEL=nightly-${CNV_PRERELEASE_VERSION}
+  CNV_SUBSCRIPTION_SOURCE=cnv-nightly-catalog-source
+fi
+
 # The kubevirt tests require wildcard routes to be allowed
 oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
 
 # Make the masters schedulable so we have more capacity to run VMs
 oc patch scheduler cluster --type=json -p '[{ "op": "replace", "path": "/spec/mastersSchedulable", "value": true }]'
 
+if [ -z "${CNV_PRERELEASE_VERSION}" ]
+  then
+  # Add pullsecret for cnv nightly channel from quay.io/openshift-cnv
+  QUAY_USERNAME=openshift-cnv+openshift_ci
+  QUAY_PASSWORD=$(jq -r .openshift_cnv_pullsecret /etc/cnv-nightly-pull-credentials)
```

Comment:
it looks like this is a typeo. It should be `if [ -n ....]` instead of `-z`. or `! [ -z ...` would work too i guess.  CI is failing because of this.

```
+ '[' -z '' ']'
+ QUAY_USERNAME=openshift-cnv+openshift_ci
++ jq -r .openshift_cnv_pullsecret /etc/cnv-nightly-pull-credentials
jq: error: Could not open file /etc/cnv-nightly-pull-credentials: No such file or directory
+ QUAY_PASSWORD=
```

Reply:
of course, fixed, thanks.